vsintegration/src/FSharp.Editor/LanguageService/FSharpProjectOptionsManager.fs

```diff
@@ -144,25 +143,23 @@ type private FSharpProjectOptionsReactor (workspace: VisualStudioWorkspaceImpl,
                             Stamp = Some(int64 (fileStamp.GetHashCode()))
                         }
 
-                cancellationToken.ThrowIfCancellationRequested()
-
                 checkerProvider.Checker.CheckProjectInBackground(projectOptions, userOpName="checkOptions")
 
                 let parsingOptions, _ = checkerProvider.Checker.GetParsingOptionsFromProjectOptions(projectOptions)
 
-                singleFileCache.[document.Id] <- (fileStamp, loadTime, parsingOptions, projectOptions)
+                singleFileCache.[document.Id] <- (fileStamp, parsingOptions, projectOptions)
 
                 return Some(parsingOptions, projectOptions)
 
-            | true, (fileStamp2, loadTime2, parsingOptions, projectOptions) ->
+            | true, (fileStamp2, parsingOptions, projectOptions) ->
                 if fileStamp <> fileStamp2 then
                     singleFileCache.Remove(document.Id) |> ignore
-                    return! tryComputeOptionsByFile document cancellationToken  (Some loadTime2)
+                    return! tryComputeOptionsByFile document
                 else
                     return Some(parsingOptions, projectOptions)
         }
     
-    let rec tryComputeOptions (project: Project) (cancellationToken: CancellationToken) =
+    let rec tryComputeOptions (project: Project) =
```

Comment:
Is the comment:
// Because this code can be kicked off before the hack, HandleCommandLineChanges, occurs,

still valid?

Reply:
That's still valid.